core/tests/test_apps.cardano.address.py

```diff
@@ -406,13 +426,26 @@ def test_shelley_address_with_byron_namespace(self):
         keychain = Keychain(node)
 
         with self.assertRaises(wire.DataError):
-            derive_base_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0, None)
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.BASE_ADDRESS,
+                address_n=[44 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0]
+            )
+            derive_address(keychain, address_parameters, 0)
 
         with self.assertRaises(wire.DataError):
-            derive_pointer_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0, CardanoCertificatePointerType(0, 0, 0))
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.POINTER_ADDRESS,
+                address_n=[44 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0],
+                certificate_pointer=CardanoCertificatePointerType(0, 0, 0)
+            )
+            derive_address(keychain, address_parameters, 0)
 
         with self.assertRaises(wire.DataError):
-            derive_enterprise_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0)
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.ENTERPRISE_ADDRESS,
+                address_n=[44 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0],
+            )
+            derive_address(keychain, address_parameters, 0)
 
```

Comment:
I think that a one-line description of the reason for raising an exception would be quite handy for each of such test cases.

Reply:
I've added a doc string at the top of the two raising test cases.